web/static.go

```diff
@@ -34,7 +34,7 @@ func (w *Web) InitStatic() {
 		mime.AddExtensionType(".wasm", "application/wasm")
 
 		staticHandler := staticFilesHandler(http.StripPrefix(path.Join(subpath, "static"), http.FileServer(http.Dir(staticDir))))
-		pluginHandler := staticFilesWithValidationHandler(http.StripPrefix(path.Join(subpath, "static", "plugins"), http.FileServer(http.Dir(*w.ConfigService.Config().PluginSettings.ClientDirectory))))
+		pluginHandler := noCacheOnNotFoundHandler(staticFilesHandler(http.StripPrefix(path.Join(subpath, "static", "plugins"), http.FileServer(http.Dir(*w.ConfigService.Config().PluginSettings.ClientDirectory)))))
```

Comment:
Given the elegance of this solution, I'm propose we just make this the default `staticFilesHandler` behaviour. We observed the original symptoms in the context of plugins, but I could imagine this problem manifesting itself for non-plugin static assets as well. I can't think of a reason for our application to ever cache the non-existence of a static asset /for an entire year/.

Reply:
Since it's been reviewed and approved would you like me to make these changes as part of the PR or a new one?  I've already got them done, just a question of process/procedure.

Reply:
@dlclark You can make the changes as part of this pr.